content/matplotlib/concepts/pyplot/terms/bar/bar.md

```diff
@@ -0,0 +1,30 @@
+# .bar()
+
+## Syntax
+A bar chart/bar graph is a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
+
+## Parameters
+plt.bar(x, height, width, bottom, align)
+
+The parameters to the function are −
+
+x       The x coordinates of the bars. 
```

Comment:
Make this a bulleted list, and wrap the parameters in inline code markdown (i.e. single backticks \`...\`) Also remove the extra spaces between the parameter and the description and replace with a single colon.